<launch>

  <!-- Launch our main node -->
  <include file="$(find msl_raptor)/launch/msl_raptor_rosbag.launch">
    <arg name="id"        value="7"           />
    <arg name="robot_type"    value="quad"  />
    <arg name="classes_names_file"  value="$(find msl_raptor)/params/objects/classes_nocs.names" /> <!-- yaml file containing all the object sizes -->
    <arg name="object_sizes_file"   value="$(find msl_raptor)/params/objects/object_params.yaml" /> <!-- yaml file containing all the object sizes -->
    <arg name="object_used_filepath" value="$(find msl_raptor)/params/objects/objects_used/" /> <!-- yaml file containing the names of objects used -->
    <arg name="object_used_file"    value="objects_used_train_nocs_1.txt" /> <!-- yaml file containing the names of objects used -->
    <arg name="experiment_param_file"    value="experiment_nocs.yaml" /> <!-- yaml file containing the names of objects used -->

    <arg name="bag_in_folder"          value="/mounted_folder/nocs/" />  <!-- path to the rosbag -->
    <arg name="bag_out_folder"         value="/mounted_folder/raptor_processed_bags" />  <!-- path to the rosbag -->
    <arg name="bag_in_name"             value="scene_1" /> <!-- name of the rosbag -->
    <arg name="b_record"            value="false" />  <!-- boolean if we are recording data in rosbag -->
    <rosparam command="load" file="$(find msl_raptor)/params/$(arg robot_type)$(arg id)_nocs.yaml" />

  </include>

</launch>
